visualization/FairnessDashboard/src/Localization/zh-TW.json

```diff
@@ -0,0 +1,154 @@
+    "backToComparisons": "多重模型檢視",
+    "outcomesTitle": "預測中的差距",
+    "minTag": "最小值",
+    "maxTag": "最大值",
+    "groupLabel": "子群組",
+    "underestimationError": "低於預測",
+    "underpredictionExplanation": "(已預測 = 0，True = 1)",
+    "overpredictionExplanation": "(已預測 = 1，True = 0)",
+    "overestimationError": "高於預測",
+    "classificationOutcomesHowToRead": "橫條圖會顯示各群組中的選擇率，表示分類為 1 的資料點。",
+    "regressionOutcomesHowToRead": "盒狀圖會顯示各群組中的預測分佈。個別資料點會重疊在上方。",
+    "classificationAccuracyHowToRead1": "橫條圖會顯示各群組中的誤差分佈。",
+    "classificationAccuracyHowToRead2": "錯誤會分割成高於預測錯誤 (當 True 標籤為 0 時，預測 1) 及低於預測錯誤 (當 True 標籤為 1 時，預測 0)。",
+    "classificationAccuracyHowToRead3": "以整體群組大小除以錯誤數目來取得回報比率。",
+    "probabilityAccuracyHowToRead1": "橫條圖會顯示各群組的平均絕對誤差，分割成高於預測及低於預測。",
+    "probabilityAccuracyHowToRead2": "在每個範例中，我們都會測量預測與標籤之間的差異。若是正數，我們稱為高於預測; 若是負數，我們稱為低於預測。",
+    "probabilityAccuracyHowToRead3": "我們回報了高於預測誤差總和及低於預測誤差總和除以整體群組大小。",
+    "regressionAccuracyHowToRead": "誤差是預測與標籤之間的差異。盒狀圖會顯示各群組中的誤差分佈。個別資料點會重疊在上方。",
+    "distributionOfPredictions": "預測分佈",
+    "distributionOfErrors": "誤差分佈",
+    "tooltipPrediction": "預測: {0}",
+    "tooltipError": "誤差: {0}"
+  },
+  "Feature": {
+    "header": "要依循哪些特徵評估模型公平性?",
+    "body": "公平性會根據模型行為中的差距部分進行評估。我們會依據各個所選特徵的值來分割資料，並評估在這些分割中，模型的成效指標與預測有何差異。",
+    "learnMore": "深入了解",
+    "summaryCategoricalCount": "此特徵有 {0} 個唯一的值",
+    "summaryNumericCount": "此數值特徵的範圍為 {0} 到 {1}，並分成 {2} 個組界。",
+    "showCategories": "全部顯示",
+    "hideCategories": "摺疊",
+    "categoriesOverflow": "   和其他 {0} 個類別",
+    "editBinning": "編輯群組",
+    "subgroups": "子群組"
+  },
+  "Metrics": {
+    "accuracyScore": "正確性",
+    "precisionScore": "精確度",
+    "recallScore": "召回率",
+    "zeroOneLoss": "0-1 損失",
+    "specificityScore": "具體程度分數",
+    "missRate": "失誤率",
+    "falloutRate": "散落率",
+    "maxError": "最大誤差",
+    "meanAbsoluteError": "平均絕對誤差",
+    "meanSquaredError": " MSE (均方誤差)",
+    "meanSquaredLogError": "平均平方對數誤差",
+    "medianAbsoluteError": "絕對誤差中位數",
+    "average": "平均預測",
+    "selectionRate": "選擇率",
+    "overprediction": "高於預測",
+    "underprediction": "低於預測",
+    "r2_score": "R 平方分數",
+    "rms_error": "RMSE (平均平方根誤差)",
+    "auc": "ROC 曲線下的區域",
+    "balancedRootMeanSquaredError": "平衡的 RMSE",
+    "balancedAccuracy": "平衡的正確性",
+    "f1Score": "F1-Score",
+    "logLoss": "Log Loss",
+    "accuracyDescription": "正確分類的資料點。",
+    "precisionDescription": "在分類為 1 的資料點當中，正確分類的資料點。",
+    "recallDescription": "在 True 標籤為 1 的資料點當中，正確分類的資料點。替代名稱: 確判為真率、敏感度。",
+    "rmseDescription": "平方誤差平均的平方根。",
+    "mseDescription": "平方誤差的平均。",
+    "meanAbsoluteErrorDescription": "誤差絕對值的平均。比 MSE 更不易受極端值影響。",
+    "r2Description": "模型所解釋的標籤變異數。",
+    "aucDescription": "從負值範例分隔出正值範例的預測品質，以分數檢視。",
+    "balancedRMSEDescription": "正值與負值範例會重新加權，使總權重相等。適用於高度不平衡的基礎資料。",
+    "balancedAccuracyDescription": "正值與負值範例會重新加權，使總權重相等。適用於高度不平衡的基礎資料。",
+    "f1ScoreDescription": "F1-Score is the harmonic mean of precision and recall."
```

Comment:
Doesn't this need a translation?

Reply:
actually this is in every language

Reply:
yes, I think they left all technical phrases untranslated for the time.